casper/src/test/scala/coop/rchain/casper/MultiParentCasperTestUtil.scala

```diff
@@ -88,15 +88,15 @@ object MultiParentCasperTestUtil {
 
   def buildGenesis(genesisParameters: Genesis): BlockMessage = {
     val storageDirectory                   = Files.createTempDirectory(s"hash-set-casper-test-genesis")
-    val storageSize: Long                  = 5 * 1024 * 1024L
-    implicit val log: Log[Task]            = Log.log[Task]
+    val storageSize: Long                  = 3024L * 1024 * 10
+    implicit val log: Log.NOPLog[Task]     = new Log.NOPLog[Task]
```

Comment:
this looks like a mis-merge

Reply:
Does it? I think it should go in anyway

Reply:
Why is the no-op log being used now?

Reply:
Also, why does the store size have to be increased?